boot/freeldr/freeldr/lib/fs/ntfs.c

```diff
@@ -1,7 +1,7 @@
 /*
  *  FreeLoader NTFS support
  *  Copyright (C) 2004  Filip Navara  <xnavara@volny.cz>
- *  Copyright (C) 2009-2010  Herv� Poussineau
+ *  Copyright (C) 2009-2010  Herv� Poussineau
```

Comment:
Please revert this change.

Reply:
Timo, This is no intentional change by sdever. It is a bug in the github website, **which always chokes on Herves** name when showing PRs. That is why I recommended in the past to change his name to H. Poussineau to workaround that github bug. You can verify my claim by putting a .patch behind the URL
https://github.com/reactos/reactos/pull/3416.patch
The change is not visible in there.

Reply:
Huh? GitHub doesn't make up changes in unmodified files!
Also what is this?
`
  *  Copyright (C) 2004  Filip Navara  <xnavara@volny.cz>
- *  Copyright (C) 2009-2010  Herv� Poussineau
+ *  Copyright (C) 2009-2010  Herv� Poussineau
  *
`

Reply:
You are right. It **is** within the patch. I have seen this exact same thing on many PRs before though.

Reply:
It's not a problem and it doesn't need a change. File was encoded in Win-1252, and now it's UTF-8.

Reply:
Thank you for double-checking this specific file @binarymaster .
We should double-check all others too with "Poussineau".